src/__tests__/to-have-style.js

```diff
@@ -116,6 +119,33 @@ describe('.toHaveStyle', () => {
     )
   })
 
+  test('handles inline customer properties', () => {
```

Comment:
What do you mean with "customer properties"? Did you mean to say "custom properties"?

```suggestion
  test('handles inline custom properties', () => {
```

Reply:
Ops, didn't see that, thanks!